### PR TITLE
Refresh return links

### LINKED ISSUE #1302: Support multiple return links
We'll need to be able to navigate to multiple levels for our state subpages. (Such as `Arizona Tests`) i.e.
```Our Data > Arizona```
instead of
`Arizona`

### PR DESCRIPTION
**Pull request by schwartzadev**
* Fixes #1302
* Moves return link styles to new file
* Supports multiple return links instead of just one
* Uses categories as return links for blog posts
* Style refresh

This points to the redesign feature branch since this was originally to support the new

* Data
  * State
    * Testing
    * Hospitalizations
    * etc.

hierarchy.

### CODE REVIEW

File: src/components/layout/header/return-links.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      // pulls the title info for nav elements
+      returnLinks[index] = topNavigation[matchingTopNavLink]
+      if (link.title) {
+        // use the specific title, if it exists
+        returnLinks[index].title = customTitle
+      }
+    }
+  })
+
+  // todo remove duplicate link locations
```
Comment (schwartzadev) [inline src/components/layout/header/return-links.js:54]: Just noticed I left this in here...I'll jump on this asap!

Comment (schwartzadev) [reply src/components/layout/header/return-links.js]: Fixed in 1e9cc45 :heavy_check_mark:

**Comment by tealtan** [comment]
Not sure if this needs additional configuration, @schwartzadev, but I was hoping for this page:
https://deploy-preview-1304--upbeat-lovelace-3e9fff.netlify.app/data/charts/us-all-key-metrics

to show `Our Data > All Charts`

**Comment by schwartzadev** [comment]
@tealtan -- fixed in 6202622421eac11a813c45c4f59c1e9f0b5dc546!

@kevee I think this is ready to review -- some of the tests are failing now, but I think we should clean that up in the feature branch instead of in this one.

**Pull request merged**